release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -516,6 +530,28 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#about-hardware-offloading_hardware-offloading[Configuring hardware offloading].
 
+[id="ocp-4-10-openstack-kuryr-resource-reduction"]
+==== Reduction of {rh-openstack} resources created by Kuryr
+
+For clusters that run on {rh-openstack}, Kuryr now only creates Neutron networks and subnets for namespaces that have at least one pod on the pods network. Additionally, pools in a namespace are only populated after at least one pod on the pods network is created in the namespace.
```

Comment:
In the second sentence, is "only" conveying the correct message?

Reply:
```suggestion
For clusters that run on {rh-openstack}, Kuryr now only creates Neutron networks and subnets for namespaces that have at least one pod on the pods network. Additionally, pools in a namespace are populated after at least one pod on the pods network is created in the namespace.
```